README.md

```diff
@@ -126,3 +126,13 @@ It should be stated that the abstraction layer (currently `NAN`) should
 support all *current* LTS releases. Given that Active LTS will overlap
 for a period of 6 months, this means that the abstraction layer will, at
 any given point in time, fully support a maximum of 2 LTS releases.
+
+## LTS Team members
+
```

Comment:
Should this include everyone in https://github.com/orgs/nodejs/teams/lts ? 

A link to the github team would be useful either way.